src/utils/command_interface.py

```diff
@@ -159,7 +159,16 @@ def command_status(self, discord_id: str) -> str:
             if not challenge:
                 result += "No current challenges"
             else:
-                result += f"User {leetcode_user_id}'s Weekly Challenge status:\n"
+                questions = self.database.table_weeklyquestion_load_by_challenge_id()
+                total = len(questions)
+                completed = 0
+                leetcode_id = user["leetcode_id"]
+                for q in questions:
+                    if LeetcodeUtil.check_challenge_completion(leetcode_id, q["title_slug"]):
+                        completed += 1
+                percentage = completed/total
```

Comment:
If `total` == 0 then you have a divide by zero error. It's possible that there are no weekly questions for a challenge, or that no challenges exist.